### PR TITLE
SnagIt for screen shot processing

### PR DESCRIPTION
**Pull request by lamagnifica**
@mhoeber and @srpearce I did some investigation into how to make our process for adding images (especially screen shots) to the guides more streamlined and consistent. I found out that @talbs and the UX crew have an updated set of edX colors in the referenced pattern library, and that SnagIt, which several of us have used in the past, has features to both save and export tool configurations which makes a consistent look easier to achieve.  
Please take a look and let me know if you have any objections to the additions I've made. If any of us prefers to continue using PhotoShop for screen shots, let me know and I'll work on getting all of the specifications for arrow heads, line widths, etc., so that you have the info to make near-identical additions.
@catong FYI

### CODE REVIEW

File: en_us/edx_style_guide/source/images.rst [hunk 1]
```diff
@@ -55,62 +66,106 @@ Keep images flush with text in bulleted and ordered lists.
...
+If you use SnagIt, note that the following .snagstyles files are included in
+the Images directory of this guide.
+
+* arrows.snagstyles
+* color_fill.snagstyles
+* outline_shapes.snagstyles
+* numbered_callouts.snagstyles
+
+To add edX SnagIt styles to your SnagIt application, drag a .snagstyles file
+onto the SnagIt icon in the toolbar of your Mac.
```
Comment (catong) [inline en_us/edx_style_guide/source/images.rst:110]: I'd just add a "results" sentence to indicate that we'll see the edx styles in edx colors added to each of the tools categories.

File: en_us/edx_style_guide/source/images.rst [hunk 2]
```diff
@@ -55,62 +66,106 @@ Keep images flush with text in bulleted and ordered lists.
...
-Instead of using text, use numbered identifiers and provide a legend in the
-text.
+Colors for Annotations
+**********************
 
-If text is used in graphics, leave 30% extra surrounding space for expansion.
+When you annotate screen shots, use the colors in the `edX pattern library`_.
+
+* For most additions, use edX Brand Secondary base, a dark pink (rgb(203, 89,
+  141), #CB598D), .
```
Comment (catong) [inline en_us/edx_style_guide/source/images.rst]: extra comma and space?

File: en_us/edx_style_guide/source/images.rst [hunk 3]
```diff
@@ -55,62 +66,106 @@ Keep images flush with text in bulleted and ordered lists.
...
+
+* If another color is required, use edX Brand Primary base, a medium blue
+  (rgb(0, 121, 188), #0079BC).
+
+Text in Annotations
+**********************
+
+When you work with screen shots, instead of using text, use numbered
```
Comment (catong) [inline en_us/edx_style_guide/source/images.rst]: "When you work with screen shots" -> "If you want to annotate screen shots"

File: en_us/edx_style_guide/source/images.rst [hunk 4]
```diff
@@ -55,62 +66,106 @@ Keep images flush with text in bulleted and ordered lists.
...
+To add edX SnagIt styles to your SnagIt application, drag a .snagstyles file
+onto the SnagIt icon in the toolbar of your Mac.
 
-Use Photoshop to edit images.
+=========================
+Border
+=========================
 
-Add a border with the following characteristics to each screen shot.
+Add a border to every image with the following characteristics.
```
Comment (catong) [inline en_us/edx_style_guide/source/images.rst]: "To every image, add a border with the following characteristics."

File: en_us/edx_style_guide/source/media_accessibility.rst [hunk 5]
```diff
@@ -62,10 +62,11 @@ driving, doing household chores, and so on, and they may not be able to focus
 on the video content at all. Try listening to only the audio track from your
 video. Is the information presented "equivalent"?
 
-.. note:: With regard to blindness, Mark feels that it is important to drive 
+.. note:: With regard to blindness, Mark feels that it is important to drive
  home the fact that if you do not vocalize unspoken visual events that are
  important to understanding the video, you will trigger a requirement to
- provide :ref:`audio descriptions<Audio Description>`_, which is both expensive and time consuming.
+ provide :ref:`audio descriptions<Audio Description>`, which is both expensive
+ and time consuming.
```
Comment (catong) [inline en_us/edx_style_guide/source/media_accessibility.rst]: I know this is not really part of this styles PR, but could we rephrase the note as a recommendation that would make sense to all readers, including open edx contributors?
"In video content, it is important to either vocalize unspoken visual events that are necessary for understanding the video or provide :ref:`audio descriptions`."

**Comment by catong** [comment]
@lamagnifica The drag to import styles works. Thanks for doing this it will be a great help for standardization and onboarding. :+1:

File: en_us/edx_style_guide/source/images.rst [hunk 6]
```diff
@@ -47,70 +54,118 @@ which narrowing the window will correct.
...
+shots. To capture part of the screen, follow these steps.
 
-#. Press CMD + SHIFT + 4. The mouse pointer changes to a crosshairs symbol.
-#. Click and drag from the upper-left corner of the area you want to capture
-   to the lower-right corner, and let go of the mouse when you've captured the
-   image you want.
+.. note:: This method does not capture the positioning of the cursor.
 
-.. note:: This method does not capture the mouse pointer.
+#. Press CMD + SHIFT + 4. The cursor changes to a cross hairs symbol.
```
Comment (srpearce) [inline en_us/edx_style_guide/source/images.rst]: crosshairs (in several online dictionaries)

File: en_us/edx_style_guide/source/images.rst [hunk 7]
```diff
@@ -47,70 +54,118 @@ which narrowing the window will correct.
...
+   you want.
 
 *****************
 Editing Images
 *****************
 
-Processing screenshots (colors for borders, overlay items) and how to make
-higher resolution version available.
+You can use any tool you like to edit images. Some guidelines for SnagIt and
+Adobe PhotoShop are included here.
```
Comment (srpearce) [inline en_us/edx_style_guide/source/images.rst]: Photoshop

File: en_us/edx_style_guide/source/images.rst [hunk 8]
```diff
@@ -47,70 +54,118 @@ which narrowing the window will correct.
...
 Annotations
 =============
 
-To annotate screen shots, use edX magenta (#b62364). If another color is
-required, use edX blue (#2276b2). 
+When you use SnagIt to add arrows, boxes, or other indicators, they are added
+as vector-based images. You can save these additions in a special SnagIt format
+so that you can edit them later: choose Save As > .snagproj format.
+
+When all of your edits are complete and you are ready to publish, Save As >
```
Comment (srpearce) [inline en_us/edx_style_guide/source/images.rst]: publish, select

File: en_us/edx_style_guide/source/images.rst [hunk 9]
```diff
@@ -47,70 +54,118 @@ which narrowing the window will correct.
...
+
+Text in Annotations
+**********************
+
+If you want to annotate screen shots, instead of adding text to the image, use
+numbered identifiers and provide a legend in the .rst file with the
+descriptions.
+
+For graphics that must include text, such as flowcharts, use a tool such as
+Adobe PhotoShop that allows you to add individually editable layers. When you
```
Comment (srpearce) [inline en_us/edx_style_guide/source/images.rst]: Photoshop

File: en_us/edx_style_guide/source/media_accessibility.rst [hunk 10]
```diff
@@ -62,10 +62,9 @@ driving, doing household chores, and so on, and they may not be able to focus
 on the video content at all. Try listening to only the audio track from your
 video. Is the information presented "equivalent"?
 
-.. note:: With regard to blindness, Mark feels that it is important to drive 
- home the fact that if you do not vocalize unspoken visual events that are
- important to understanding the video, you will trigger a requirement to
- provide :ref:`audio descriptions<Audio Description>`_, which is both expensive and time consuming.
+.. note:: In video content, it is important to either vocalize unspoken visual
```
Comment (srpearce) [inline en_us/edx_style_guide/source/media_accessibility.rst]: it is important to --> you must

**Comment by lamagnifica** [comment]
@srpearce and @mhoeber , I'm not actually looking for a dev edit, copy edit, or sanity check in this PR. I'm interested in whether you think that moving toward increased use of SnagIt, by all or some of the team, is worth doing for the increase in consistency.

**Comment by srpearce** [comment]
Yes, it sounds like it could be a good idea.

**Comment by mhoeber** [comment]
sounds good.  I for one wouldn't mind a quick demo at some point.

**Comment by lamagnifica** [comment]
git is a horrible tool

**Pull request closed without merging**